src/StringUtils.vala

```diff
@@ -36,4 +36,14 @@ public class FeedReader.StringUtils {
 	{
 		return string.joinv(sep, l.to_array());
 	}
+
+	public static Gee.List<string> encapsulate(Gee.List<string> l, string encapsulation)
+	{
+		for(int i = 0; i < l.size; i++)
+		{
+			l[i] = encapsulation + l[i] + encapsulation;
```

Comment:
If this is for quoting SQL strings, we need to also double all single quotes:

> A string constant is formed by enclosing the string in single quotes ('). A single quote within the string can be encoded by putting two single quotes in a row - as in Pascal. C-style escapes using the backslash character are not supported because they are not standard SQL.

(Also consider moving this to a static function on the Sqlite class)

```
public static string quote_string(string str)
{
    var escaped = str.replace("'", "''");
    return @"'$(escaped)'";
}
```